### PR TITLE
Check server on app start and timed intervals

### LINKED ISSUE #44: Manage non-logged app
Redirect to login or if not possible, show alert.

### PR DESCRIPTION
**Pull request by tokland**
Closes #44 

Checks on app bootup and every 10 seconds.

### CODE REVIEW

**Comment by tokland** [comment]
not_logged_in

File: src/app.js [hunk 1]
```diff
@@ -35,6 +35,17 @@ render(
   </MuiThemeProvider>, 
   document.getElementById('app'));
 
+function safeGetUserSettings() {
+    const redirect = (err) => {
+        window.location.assign(config.loginUrl);
```
Comment (adrianq) [inline src/app.js]: @tokland we have this redirection to loginUrl here and in redirectToLogin method. The url path and the code that makes the redirection is redundant. Can we maybe extract this redirection method into an utils function? Then in the sessiondialog, redirectToLogin doesn't need to be a method hardcoded but a property that we pass to the component SessionDialog.  What do u think? Maybe I am being too picky, if so just let me know :)

**Comment by adrianq** [comment]
@tokland just a minor comment...

Also for the record: we are finally checking the app every 15 mins and it is more a 'keep me alive' feature than an actual validation. We will have to adjust this number depending on the time out in the dhis2 instance of the client

**Comment by adrianq** [comment]
@tokland Sorry to bother you again. I am merging this PR as  I am about to release a new version and I think it would be good to have this feature on it. I have created a [card](https://github.com/EyeSeeTea/dataset-configuration/issues/58) to discuss this minor refactor I mentioned in my review. If finally we decide to go for it we can implement it as part of the next sprint

**Pull request merged**